sql/updates/world/master/yyyy_mm_dd_xx_world_temple_of_kotmogu.sql

```diff
@@ -0,0 +1,277 @@
+SET @CGUID := 20000000000;
+SET @OGUID := 20000000000;
+SET @SPAWNGROUP := 200000;
+SET @ATPROPERTIESID := 200000000; -- CHANGE ME IN SCRIPT
+SET @ATID := 200000000;
+SET @ATSPAWNID := 200000000;
+
+DELETE FROM `battleground_template` WHERE `ID`=699;
+INSERT INTO `battleground_template` (`ID`, `AllianceStartLoc`, `HordeStartLoc`, `Weight`, `Comment`) VALUES
+(699, 4059, 4060, 1, 'Temple of Kotmogu');
+
+DELETE FROM `battleground_scripts` WHERE `MapId`=998;
+INSERT INTO `battleground_scripts` (`MapId`, `BattlemasterListId`, `ScriptName`) VALUES
+(998, 0, 'battleground_temple_of_kotmogu');
+
+DELETE FROM `areatrigger_template` WHERE (`IsCustom`=0 AND `Id` IN (20901)) OR (`Id` BETWEEN @ATID+0 AND @ATID+1 AND `IsCustom` = 1);
+INSERT INTO `areatrigger_template` (`Id`, `IsCustom`, `Flags`, `VerifiedBuild`) VALUES
+(20901, 0, 0, 56008),
+(@ATID+0, 1, 0x1, 0),
+(@ATID+1, 1, 0x1, 0);
+
+DELETE FROM `areatrigger_create_properties` WHERE (`IsCustom`=1 AND `Id` BETWEEN @ATPROPERTIESID+0 AND @ATPROPERTIESID+2);
+INSERT INTO `areatrigger_create_properties` (`Id`, `IsCustom`, `AreaTriggerId`, `IsAreatriggerCustom`, `Flags`, `MoveCurveId`, `ScaleCurveId`, `MorphCurveId`, `FacingCurveId`, `AnimId`, `AnimKitId`, `DecalPropertiesId`, `TimeToTarget`, `TimeToTargetScale`, `Shape`, `ShapeData0`, `ShapeData1`, `ShapeData2`, `ShapeData3`, `ShapeData4`, `ShapeData5`, `ShapeData6`, `ShapeData7`, `SpellForVisuals`, `ScriptName`, `VerifiedBuild`) VALUES
+(@ATPROPERTIESID+0, 1, 20901, 0, 0, 0, 0, 0, 0, -1, 0, 0, 0, 0, 0, 4, 4, 0, 0, 0, 0, 0, 0, 294534, 'at_bg_temple_of_kotmogu_healing_buff', 56008), -- 294534 (PvP Rune Rejuv Visual) (Area: Temple of Kotmogu - Difficulty: 0)
+(@ATPROPERTIESID+1, 1, @ATID+0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 33, 47, 5, 33, 47, 5, 0, 0, 0, 'at_bg_temple_of_kotmogu_small_area', 0),
+(@ATPROPERTIESID+2, 1, @ATID+1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 3, 50, 50, 0, 0, 0, 0, 0, 0, 0, 'at_bg_temple_of_kotmogu_medium_area', 0);
```

Comment:
My recommendation would be to use the same entry for both Id and AreaTriggerId in case we lack a spell to look properties on, since we got IsCustom, there shouldn't be any issues to identify them as made-up. 

In any case, did you get any entry in a sniff for the two later? I see that you've got very specific shapes, which could indicate these came from a sniff, which *usually* also comes with an Entry.

Reply:
The last two are serverside areatriggers, with the parameters being done manually. I don't know how else it should've been done. When you enter these area's you get aura, when you leave said area, its removed. There is no area-entry for these

Reply:
![image](https://github.com/user-attachments/assets/7ead9c31-d501-4c4c-9f4c-b70ee45c6eb6)
posting here